mobile-widgets/qml/Settings.qml

```diff
@@ -28,55 +28,49 @@ Kirigami.ScrollablePage {
 			title: qsTr("General settings")
 			isExpanded: true
 
-		}
-
-		GridLayout {
-			id: cloudSetting
-			columns: 3
-
-			TemplateLabel {
-				text: qsTr("Cloud status")
-				font.pointSize: subsurfaceTheme.headingPointSize
-				font.weight: Font.Light
-				Layout.topMargin: Kirigami.Units.largeSpacing
-				Layout.bottomMargin: Kirigami.Units.largeSpacing / 2
-				Layout.columnSpan: 3
-			}
-			TemplateLabel {
-				text: qsTr("Email")
-				Layout.preferredWidth: gridWidth * 0.15
-			}
-			TemplateLabel {
-				text: PrefCloudStorage.cloud_verification_status === CloudStatus.CS_NOCLOUD ? qsTr("Not applicable") : PrefCloudStorage.cloud_storage_email
-				font.pointSize: subsurfaceTheme.regularPointSize
-				Layout.preferredWidth: gridWidth * 0.60
-			}
-			SsrfButton {
-				id: changeCloudSettings
-				text: qsTr("Change")
-				onClicked: {
-					PrefCloudStorage.cloud_verification_status = CloudStatus.CS_UNKNOWN
-					manager.startPageText  = qsTr("Starting...");
+			GridLayout {
+				id: cloudSetting
+				visible: sectionGeneral.isExpanded
+				columns: 3
+
+				TemplateLabel {
+					text: qsTr("Cloud status")
+					font.pointSize: subsurfaceTheme.headingPointSize
+					font.weight: Font.Light
+					Layout.topMargin: Kirigami.Units.largeSpacing
+					Layout.bottomMargin: Kirigami.Units.largeSpacing / 2
+					Layout.columnSpan: 3
+				}
+				TemplateLabel {
+					text: qsTr("Email")
+					Layout.preferredWidth: gridWidth * 0.15
+				}
+				TemplateLabel {
+					text: PrefCloudStorage.cloud_verification_status === Enums.CS_NOCLOUD ? qsTr("Not applicable") : PrefCloudStorage.cloud_storage_email
```

Comment:
funny thing... your commit messages says "without changing anything", but here an `Enums` change snuck in, which depends on the other PR that is still WIP... :-(